api/v3/Contact.php

```diff
@@ -618,7 +618,6 @@ function _civicrm_api3_greeting_format_params($params) {
 
     $nullValue = FALSE;
     $filter = array(
-      'contact_type' => $params['contact_type'],
```

Comment:
enotice fix when not set + this is not really the place to worry about filtering